skimage/exposure/exposure.py

```diff
@@ -37,6 +39,9 @@ def histogram(image, nbins=256):
     nbins : int
         Number of bins used to calculate histogram. This value is ignored for
         integer arrays.
+    multichannel : bool, optional
+        If True, calculates the histogram for all the channels using the same
```

Comment:
I would say "for each channel", instead of "for all". It reinforces the fact that each channel il taken independently.